src/js/modules/supervisors/FileManager.ts

```diff
@@ -8,7 +8,7 @@
  * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
  */
 
-import * as Inotify from "inotifywait";
```

Comment:
Leaving a comment about the commit message. The message says your updating mocks for chokidar library on tests, but here are changes in the actual source (not unit tests) that replace the inotify impl with chokidar. I would rename the commit to be something like "coproc/js: Swap out the InotifyWait dependency in favor of chokidar"

Reply:
Got it